crates/driver/src/infra/api/error.rs

```diff
@@ -19,6 +19,7 @@ enum Kind {
     MissingSurplusFee,
     InvalidTokens,
     InvalidAmounts,
+    InvalidOrders,
```

Comment:
The naming should be consistently `NoValidOrders` as `InvalidOrders` implies the existence of orders that are invalid instead of allowing for 0 orders.